third_party/move/move-compiler-v2/tests/reference-safety/v1-borrow-tests/writeref_borrow_invalid.old.exp

```diff
@@ -3,7 +3,7 @@
 
 Diagnostics:
 bug: bytecode verification failed with unexpected status code `WRITEREF_EXISTS_BORROW_ERROR`. This is a compiler bug, consider reporting it.
```

Comment:
Just double-checking: we're not worried about bugs in .old.exp, right?

Reply:
Yes, that's my understanding: note that this code failed bytecode verification before (and continues to do so).